Makefile

```diff
@@ -141,16 +141,40 @@ bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metada
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	cd config/default && $(KUSTOMIZE) edit set image rbac-proxy=$(RBAC_PROXY_IMG)
 	cd config/console && $(KUSTOMIZE) edit set image odf-console=$(ODF_CONSOLE_IMG)
+
+ifeq ($(FUSION) , true)
+	@echo -e "\nBuilding IBM Fusion Data Foundation bundle"
+	mv bundle.Dockerfile odf.bundle.Dockerfile
+	cd config/fusion/bases && $(KUSTOMIZE) edit add annotation --force 'olm.skipRange':"$(SKIP_RANGE)" && \
+	        $(KUSTOMIZE) edit add annotation --force 'operators.operatorframework.io/operator-type':"$(OPERATOR_TYPE)" && \
+		$(KUSTOMIZE) edit add patch --name fdf-operator.v0.0.0 --kind ClusterServiceVersion\
+		--patch '[{"op": "replace", "path": "/spec/replaces", "value": "$(REPLACES)"}]'
+	$(KUSTOMIZE) build config/fusion | $(OPERATOR_SDK) generate bundle -q --overwrite  --version $(VERSION) \
+	$(BUNDLE_METADATA_OPTS) --output-dir fusion/bundle --kustomize-dir config/fusion \
+	--package fdf-operator
+	cp bundle/metadata/dependencies.yaml fusion/bundle/metadata/dependencies.yaml
```

Comment:
we are required to make changes to the dependencies.yaml as well. We need to replace the ocs-operator to fcs-operator.